add sql statement about pump and drainer (#2945)
pingcap/docs: sql-statements/sql-statement-show-master-status.md

---
title: SHOW MASTER STATUS
summary: An overview of the usage of SHOW MASTER STATUS for the TiDB database.
category: reference
---

# SHOW MASTER STATUS

> `SHOW MASTER STATUS` statements displays the latest TSO in the cluster.

Reviewer: ```suggestion
The `SHOW MASTER STATUS` statement displays the latest TSO in the cluster.
```

> ## 示例

Reviewer: ```suggestion
## Examples
```

{{< copyable "sql" >}}

```sql
SHOW MASTER STATUS;
```

```sql
+-------------+--------------------+--------------+------------------+-------------------+
| File        | Position           | Binlog_Do_DB | Binlog_Ignore_DB | Executed_Gtid_Set |
+-------------+--------------------+--------------+------------------+-------------------+
| tidb-binlog | 416916363252072450 |              |                  |                   |
+-------------+--------------------+--------------+------------------+-------------------+
1 row in set (0.00 sec)
```

## MySQL compatibility

> The output of `SHOW MASTER STATUS` is designed to match MySQL. However, the execution results are different, with binlog location information in MySQL and the latest TSO information in TiDB.

Reviewer: ```suggestion
The output of `SHOW MASTER STATUS` is designed to match MySQL. However, the execution results are different in that the MySQL result is the binlog location information and the TiDB result is the latest TSO information.
```
